frontend/src/shared/layout/horizontal-line.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License").
+  You may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import { ColumnLayout } from '@cloudscape-design/components';
+import React from 'react';
+
+
+export const HorizontalLine = () => {
```

Comment:
Why are we adding this? This is kind of a weird way to effectively create a `<hr>` that we don't have any control over. I think creating an actual `<hr>` would be preferable, but I am confused why we need this in the first place.

PR looks good to me otherwise

Reply:
`<hr>` is a normal html horizontal line without styling while this component is a styled horizontal line that matches the Cloudscape UI design. Unfortunately Cloudscape does not have a HorizontalLine component that exposes their styling, but this is a hacky way to obtain the same result.